settings.py

```diff
@@ -2222,3 +2221,5 @@ def _determine_couch_databases(couch_databases):
             'NAME': 'custom.nic_compliance.password_validation.UsedPasswordValidator',
         }
     ]
+
+PACKAGE_MONITOR_REQUIREMENTS_FILE = os.path.join(FILEPATH, 'requirements', 'requirements.txt')
```

Comment:
I think this could go in localsettings as well

Reply:
so the thing is this relies on FILEPATH which is defined in settings.  I didn't see any example of how you could access FILEPATH from localsettings

Reply:
I think it should auto add it because earlier in the file we have `from localsettings import *`

Reply:
(resolved)